README.md

```diff
@@ -12,8 +12,8 @@ using [RServe](https://www.rforge.net/Rserve/doc.html).
 ### Generic R
 [![Maven metadata URI](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/com/feedzai/openml-generic-r/maven-metadata.xml.svg)](https://mvnrepository.com/artifact/com.feedzai/openml-generic-r)
 
-You can find in the `openml-generic-r` module a provider that allows
-users to load in code that conforms to a simple API.
+The `openml-generic-r` module contains a provider that allows
+developers to load code that conforms to a simple API.
```

Comment:
maybe update to `developers to load R code` ? It's implicit that the code loaded should be in the R programming language, but better explicit than implicit, do you agree?